mppm/troubleshoot/debug-mppm.md

```diff
@@ -6,6 +6,8 @@ description: Learn how to test a player running in Multiplayer Play Mode
 
 Use the Test Runner included in the [Unity Test Framework (UTF)](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/index.html) to test players in a multiplayer project.
 
+Note: Multiplayer Play Mode is compatibly with Unity Editor versions 2023.1 or later.
+
```

Comment:
Again maybe we should note mppm 1.1.0 (latest) minimum editor version is now 6000.0.3f1

Reply:
See other reply.

Reply:
Will add this update to 1.1.0 when released :)